src/models/Task.ts

```diff
@@ -1,11 +1,13 @@
 import { Timestamp } from 'firebase/firestore';
+import { DocumentReference } from 'firebase/firestore';
 
 export interface Task {
   name: string;
   description: string;
   importance: number;
   deadline: Timestamp;
-  active: boolean;
+  archived: boolean;
   status: boolean;
   uuid: string;
```

Comment:
BTW. As I understand it is a user's id. If so it's better to use `userId` or `user_id` naming as `uuid` is a name of a widely used [library](https://www.npmjs.com/package/uuid) in js world, so `uuid` is ambiguous.

Reply:
ok, will fix everything now